chall01/krusthol.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
+import sys
+import string
+
+def bad_character(str):
+    good = set(string.ascii_lowercase + ' ')
+    for c in str:
+        if c not in good:
+            return True
+    return False
+
+if len(sys.argv) != 2:
+    sys.exit("usage: ./krusthol.py <a-zA-Z string>")
+arg = sys.argv[1].lower()
+if len(arg) == 0 or bad_character(arg):
+    sys.exit("usage: ./krusthol.py <a-zA-Z string>")
+for c in arg:
+    if c == 'a':
+        sys.stdout.write('.-')
```

Comment:
In python3 you can use print statement without a newline:
`print('.-', end='')`